kuma/javascript/src/ga-provider.jsx

```diff
@@ -6,25 +6,26 @@ export type GAFunction = (...any) => void;
 
 export const CATEGORY_MONTHLY_PAYMENTS = 'monthly payments';
 
-const GA_QUERY_KEY = 'ga';
+const GA_SESSION_STORAGE_KEY = 'ga';
 
-const QUERY_PARAM_GA_DATA = {
-    'subscription-success': {
-        hitType: 'event',
-        eventCategory: CATEGORY_MONTHLY_PAYMENTS,
-        eventAction: 'successful subscription',
-        eventLabel: 'subscription-landing-page',
-    },
-    'banner-cta': {
-        hitType: 'event',
-        eventCategory: CATEGORY_MONTHLY_PAYMENTS,
-        eventAction: 'subscribe intent',
-        eventLabel: 'banner',
-    },
-};
+function getPostponedEvents() {
+    return JSON.parse(
+        sessionStorage.getItem(GA_SESSION_STORAGE_KEY) || JSON.stringify([])
```

Comment:
What's wrong with 
```suggestion
        sessionStorage.getItem(GA_SESSION_STORAGE_KEY) || '[]'
```
I'm not trying to change your mind; just trying to learn from you.

Reply:
I like the former better because it seems more resistant to typos. The latter is "just some string which happens to be the serialized form of an array". I guess this example doesn't drive the point home, but I like erring using the (de-)serialization functions for even the simplest of cases, to not create precedence.